WORKSPACE

```diff
@@ -8,12 +8,75 @@ tf_configure(
 
 http_archive(
     name = "org_tensorflow",
-    sha256 = "c030cb1905bff1d2446615992aad8d8d85cbe90c4fb625cee458c63bf466bc8e",
-    strip_prefix = "tensorflow-2.12.0",
+    sha256 = "ce357fd0728f0d1b0831d1653f475591662ec5bca736a94ff789e6b1944df19f",
+    strip_prefix = "tensorflow-2.14.0",
     urls = [
-        "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.12.0.tar.gz",
+        "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.14.0.tar.gz",
     ],
 )
+# TODO: please double check what it is really required or not in this section
```

Comment:
@seanpmorgan This was commited by the bot/proxy GitHub account in TF `WORKSPACE`. Can you ping SIG-Build team to know who is the owner of that new section?

Reply:
@yongtang is handling the same at https://github.com/tensorflow/io/pull/1831